build/yamls/externalnode/conf/antrea-agent.conf

```diff
@@ -0,0 +1,43 @@
+# FeatureGates is a map of feature names to bools that enable or disable experimental features.
+featureGates:
+# Enable running agent on an unmanaged VM/BM.
+  ExternalNode: true
+
+# Enable Antrea ClusterNetworkPolicy feature to complement K8s NetworkPolicy for cluster admins
+# to define security policies which apply to the entire cluster, and Antrea NetworkPolicy
+# feature that supports priorities, rule actions and externalEntities in the future.
+  AntreaPolicy: true
+
+# Enable collecting and exposing NetworkPolicy statistics.
+  NetworkPolicyStats: true
+
+# Name of the OpenVSwitch bridge antrea-agent will create and use.
+# Make sure it doesn't conflict with your existing OpenVSwitch bridges.
+#ovsBridge: br-int
+
+# Datapath type to use for the OpenVSwitch bridge created by Antrea. Supported values are:
+# - system
+# - netdev
+# 'system' is the default value and corresponds to the kernel datapath. Use 'netdev' to run
+# OVS in userspace mode (not fully supported yet). Userspace mode requires the tun device driver to
+# be available.
+#ovsDatapathType: system
+
+# The port for the antrea-agent APIServer to serve on.
+# Note that if it's set to another value, the `containerPort` of the `api` port of the
+# `antrea-agent` container must be set to the same value.
+#apiPort: 10350
+
+# NodeType is type of the Node where Antrea Agent is running.
+# Defaults to "k8sNode". Valid values include "k8sNode", and "externalNode".
+nodeType: externalNode
+
+# The path to access the kubeconfig file used in the connection to K8s APIServer. The file contains the K8s
+# APIServer endpoint and the token of ServiceAccount required in the connection.
+clientConnection:
+  kubeconfig: antrea-agent.kubeconfig
+
+# The path to access the kubeconfig file used in the connection to Antrea Controller. The file contains the
+# antrea-controller APIServer endpoint and the token of ServiceAccount required in the connection.
+antreaClientConnection:
+  kubeconfig: antrea-agent.antrea.kubeconfig
```

Comment:
@wenyingd should we use helm to generate this file based on template? Otherwise we would need to update this file manually whenever there is a change to related parameters? I don't mean it needs to be done in this PR.